xskillscore/core/np_deterministic.py

```diff
@@ -60,7 +62,7 @@ def _check_weights(weights):
     if weights is None:
         return None
     # catch if np.ndarray values are None
-    elif (weights == None).all():
+    elif (weights is None).all():
```

Comment:
I think the CI is failing with `is` instead of `==`
https://travis-ci.com/github/raybellwaves/xskillscore/builds/174882542#L1027

Reply:
I'll put it back. `flake8` had asked me to change it.